ohapi/public.py

```diff
@@ -168,3 +168,34 @@ def download(source=None, username=None, directory='.', max_size='128m',
         for value in executor.map(download_url_partial, results):
             if value:
                 logging.info(value)
+
+
+def get_data_members_by_source(base_url=BASE_URL_API):
```

Comment:
minor comment: the "data" in here doesn't attach to "members" in the API URL, it's a separate level, "public-data".

The function should be `get_members_by_source`, to be consistent with the API endpoint `members-by-source`.